app/controllers/comments_controller.rb

```diff
@@ -103,4 +119,13 @@ def send_notifications
       send_notification('%<poster>s has also commented on submission %<target>s.', user_id)
     end
   end
+
+  private
```

Comment:
This `private` is redundant because there is already one before `comment_params`.